src/test/java/io/github/epam/test/data/UserUtils.java

```diff
@@ -3,5 +3,8 @@
 import io.github.com.entities.User;
 
 public class UserUtils {
-    public static User DEFAULT_USER = new User();
+    public static final User DEFAULT_USER = new User().set(u -> {
+        u.userName = "default";
```

Comment:
Remove user's credentials from the code. 
They are passed as command line parameters of the Maven command: "mvn test -Dreport.portal.user=$TEST_USER_LOGIN -Dreport.portal.password=$TEST_USER_PASSWORD" (see travis.yml to understand current Travis CI build)